src/wp-admin/includes/ajax-actions.php

```diff
@@ -1291,7 +1291,7 @@ function wp_ajax_replyto_comment( $action ) {
 	if ( empty( $post->post_status ) ) {
 		wp_die( 1 );
 	} elseif ( in_array( $post->post_status, array( 'draft', 'pending', 'trash' ), true ) ) {
-		wp_die( __( 'Error: You cannot reply to a comment on a draft post.' ) );
+		wp_die( __( '<strong>Error:</strong> You cannot reply to a comment on a draft post.' ) );
```

Comment:
The error handler doesn't seem to support HTML here: https://github.com/WordPress/wordpress-develop/blob/4558de543c64c877ffea4cdfbdf67d743c3e4391/src/js/_enqueues/admin/edit-comments.js#L1186

Reply:
Can we make that error handle support HTML or is this one error message that has to be w/o the error tag?

Reply:
Another idea, fixing it right now.

Reply:
Better to remove the `strong` tag completely and/or rephrase a bit the sentence :)

Reply:
Rephrase it is. Since otherwise we have almost all our strings that start with "Error" with a strong tag and only these few not.  For consistency sake.